### ISSUE TITLE
Add expand/collapse all to sections of dottydoc

### ISSUE THREAD

**Issue by felixmulder** [lampepfl/dotty#2211]
We currently have `[+]` to expand items with longer comments. I want to have an “expand/collapse all” button at the top left of each category where this is relevant.
So for example if you look here: http://dotty.epfl.ch/api/dotty/tools/dotc/transform/IsInstanceOfEvaluator.html
You can see that the member phaseName has an expand button. I would like its category (i.e. “Members”) to have a collapse all button.

So before this change you would have:
<img width="1164" alt="screen shot 2017-04-10 at 10 45 53" src="https://cloud.githubusercontent.com/assets/1530049/24860758/651901b0-1df6-11e7-9a70-d6d1f971640c.png">

And after you’d have:
<img width="1167" alt="screen shot 2017-04-10 at 10 47 15" src="https://cloud.githubusercontent.com/assets/1530049/24860763/6c7ab200-1df6-11e7-9688-02c3711d15ed.png">

But with the `[+]` on “Members” having the same style as the one inside the categories
This would be the first superficial change - which means just changing the HTML

After this, I want one more change: collapsed comments should contain nothing. Currently they contain an abbreviated version of the docstring, I don’t think it is beneficial. It clutters a lot and in the end you know roughly what you’re looking for if you’re browsing the API.

In summary:
- Give each relevant category (e.g. “Members”, “Constructors”) a “expand/collapse all” `[+]`-button
- Remove the summary on collapsed
- Remove summary-related code in the dottydoc scala codebase

**Comment by felixmulder** [comment]
Assigned to @hhandoko

**Comment by Varunram** [comment]
@felixmulder I'm kinda new here so pardon me but is @hhandoko still working on this? If not, can I work on this? Thanks

**Comment by felixmulder** [comment]
There hasn't been any activity from Herdy on this for a while - I say go ahead! :)

**Comment by hhandoko** [comment]
@felixmulder Sorry this fell by the wayside...

**Comment by felixmulder** [comment]
@hhandoko - no worries mate, life happens =)
